pkg/rpc/client/wsclient.go

```diff
@@ -398,3 +402,9 @@ func (c *WSClient) UnsubscribeAll() error {
 	}
 	return nil
 }
+
+// IsConnectionClosed tells whether WS connection was closed. If so, then no WS
+// requests can be performed and no responses will be received.
+func (c *WSClient) IsConnectionClosed() bool {
```

Comment:
I doubt it's useful. If `WSClient` has not errored out (like in https://github.com/nspcc-dev/neo-go/issues/2421#issuecomment-1088589865), it's assumed to be active. Any request to it can fail anyway for a number of reasons and any _local_ status check is almost useless, a real request needs to be sent.

Reply:
Dropped the commit.